lndgrpc/client.py

```diff
@@ -147,16 +242,16 @@ def describe_graph(self):
         return response
 
     @handle_rpc_errors
-    def get_channel_info(self, channel_id):
+    def get_chan_info(self, channel_id):
```

Comment:
I don't think we should break existing method names. Lets leave it as is.

Reply:
can we keep both and put a deprecation warning maybe? I just want to follow the docs exactly to not have to think about it